### ISSUE TITLE
Ensure links are distinguished from surrounding text in a way that does not rely on color. : A11y_A11y_Awesome AZD_AI4W

### ISSUE THREAD

**Issue by Mana-swi** [Azure/awesome-azd#818]
### Tags

#A11yTCS; #A11ySev2; #A11yWCAG; #WCP; #ChromiumEdge; #FTP; #Win11; #awesome-azd-wcp; #Feature_AwesomeAZD_Web_Mar2024; #DesktopWeb;#CELA P3;#link-in-text-block;#MAS1.4.1;#Use of color;#ES Chat;#copilot:agent=A11yAgent;

 **"Leveraging SWE Agent can help accelerate this accessibility bug fix; review and validate all AI generated changes before merge."** 

**"Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Feng.ms%2Fdocs%2Fcoreai%2Fdevdiv%2Fone-engineering-system-1es%2F1es-docs%2Fes-chat%2Findex&data=05%7C02%7Cv-mchougule%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838215421556%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=%2B1yeHqbNhHcMuQaZNXDo%2B%2FiWxH0mZFSmWpamQk2ftws%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchannel%2F19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2FCAI%2520Accessibility%2520Support%2520Channel%3FgroupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7C02%7Cv-mchougule%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838215450064%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=yncZXSKXaJlQ3HpXeR3TK03MNIhUe8tV2yZl49clFZY%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Foutlook.office.com%2Fbook%2Fmicrosoft2779007%2540microsoft.onmicrosoft.com%2Fs%2FbLeGBltMW0qI0WRLsTqx2A2%3Fismsaljsauthenabled&data=05%7C02%7Cv-mchougule%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838215467554%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=i4Fn6qPJw4eg3Ny6En0%2FPnIqw%2BpJpHuT%2F%2BZH2hE%2FRTA%3D&reserved=0)."**

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:
Application: Awesome AZD
URL: https://azure.github.io/awesome-azd/docs/contribute/
Chromium Edge: Version 147.0.3912.60 (Official build) (64-bit)

### **To Reproduce**
1. Go to 'https://azure.github.io/awesome-azd/docs/contribute/'
2. Run the accessibility insights for web -fastpass
3. See errors.

### Issue:
Ensure links are distinguished from surrounding text in a way that does not rely on color.

### Path:
1. p:nth-child(7) > a[target="_blank"][rel="noopener noreferrer"]
2. p:nth-child(19) > a[target="_blank"][rel="noopener noreferrer"]

### Snippet: 

1.  <a href="https://github.com/Azure/awesome-azd/issues?q=is%3Aopen+is%3Aissue+label%3Arequested-contribution" target="_blank" rel="noopener noreferrer" class="">requested templates</a>
2. <a href="https://learn.microsoft.com/azure/developer/azure-developer-cli/make-azd-compatible?source=recommendations&amp;pivots=azd-create#azd-conventions" target="_blank" rel="noopener noreferrer" class="">by azd conventions</a>

### Related Paths:
- p:nth-child(7)
- p:nth-child(19)

### URLs:
- https://azure.github.io/awesome-azd/docs/contribute/
- https://azure.github.io/awesome-azd/docs/faq/

### How to fix:
Fix the following:

-   The link has insufficient color contrast of 2.81:1 with the surrounding text. (Minimum contrast is 3:1, link text: #0369a1, surrounding text: #1c1e21)
- The link has no styling (such as underline) to distinguish it from the surrounding text

### Screenshots:

<img width="1354" height="704" alt="Image" src="https://github.com/user-attachments/assets/9d8e47c7-1cf7-4b32-8037-deb704630023" />

<img width="1356" height="684" alt="Image" src="https://github.com/user-attachments/assets/18564152-4578-4335-95f6-1ce9cf4519ba" />

**Comment by Mana-swi** [comment]
#Reopened; #Regressed:04-21-26;

@hemarina 
Verified the issue in below environment
URL:(https://azure.github.io/awesome-azd/)](https://azure.github.io/awesome-azd/docs/contribute/)


The issue is still repro'ing, hence reactivating the bug.

<img width="1365" height="689" alt="Image" src="https://github.com/user-attachments/assets/8c175c18-2ebd-4c81-b5b7-73dc38c44d10" />

**Comment by hemarina** [comment]
The site is not released until tomorrow. Please check tomorrow.
